tests/network/openvpn_client.pm

```diff
@@ -0,0 +1,87 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test OpenVPN on two machines. This one is client.
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use base 'consoletest';
+use testapi;
+use lockapi;
+use mm_network;
+use utils qw(systemctl zypper_call exec_and_insert_password);
+use strict;
+
+sub run {
+    my ($self) = @_;
+    select_console "root-console";
+
+    # Configure static network, disable firewall
+    configure_default_gateway;
```

Comment:
there are :
1. `setup_static_network` in network_utils  which doing almost the same. With small difference which may or may not be fine for you but worth to look anyway 
2. `setup_static_mm_network` in y2x11test which doing **exactly** same thing you can reuse it 
let's not increase duplicate code in repo

Reply:
I implemented the `setup_static_mm_network` from `y2x11test` so you can see [server](http://pdostal-server.suse.cz/tests/222#) and [client](http://pdostal-server.suse.cz/tests/223).